### PR TITLE
Game feature 1.0, including a Game class and Player class.

### PR DESCRIPTION
**Pull request by RaykenJ**
Player class still need to be modified for makeDecision method

### CODE REVIEW

File: src/main/java/Game.java [hunk 1]
```diff
@@ -1,22 +1,211 @@
...
+        String decision;  // the decision made by current player
+        int call;  // current highest bidding
+
+        currentCall[0] = players[0].betSmallBlind();
+        call = currentCall[0];
+        pool.addMoney(players[0], currentCall[0]);
+
+        currentCall[1] = players[1].betBigBlind(call);
+        call = currentCall[1];
+        pool.addMoney(players[1], currentCall[1]);
```
Comment (enfz) [inline src/main/java/Game.java:28]: Probably will use a pool Use Case instead

**Review by enfz — APPROVED** [review]
The code needs Javadoc comments and clean architecture implementation as you said. But otherwise, the logic and implementation look good. Presenter/Controller probably needed to update the view of GameScreen and take the inputs from GameScreen.

**Review by ericshi03 — APPROVED** [review]
Logic wise I can see what your code is trying to do. It might be better to split up some parts of your main method a bit further but otherwise it looks good. Aside from implementing clean architecture your code is good.

**Review by Jefferson-liu — APPROVED** [review]
The code for the game class looks good, but we will have to implement clean architecture in the future. Also, there is a lack of tests right now, so probably add those. Make sure that this implementation of the game class works with the other implementation, but other than that, the logic looks fine. Assuming we won't have issues merging the two game classes, the code is good.

**Review by xiaoxinamek — APPROVED** [review]
The game class are intuitive and easy to understand.  I like the logic, but it seems dealing with lots of tasks at the same time, I wonder if it is better we breaks it into different stages. Or maybe this would not be a problem after refactory. I think your code is good.

**Pull request merged**